community/kernel/src/test/java/org/neo4j/kernel/impl/transaction/state/DefaultIndexProviderMapTest.java

```diff
@@ -44,7 +43,7 @@ public void shouldNotSupportMultipleProvidersWithSameDescriptor()
         // when
         try
```

Comment:
Since you already started improving the code, replace it with `assertThrows()` as well